rubin_scheduler/scheduler/features/features.py

```diff
@@ -182,6 +182,8 @@ def __init__(self, scheduler_note=None, bandname=None, note=None):
         if self.scheduler_note == "":
             self.scheduler_note = None
         self.bandname = bandname
+        # Will not add filtername backup here; users should reach
+        # down as far as the features in configurations
```

Comment:
Maybe a missing "not" in here? Should be something like "users do not set feature.__init__ kwargs when configuring a scheduler obejct"